src/Cooler.sol

```diff
@@ -321,6 +321,8 @@ contract Cooler is Clone {
         // Update the load lender and the recipient.
         loans[loanID_].lender = msg.sender;
         loans[loanID_].recipient = msg.sender;
+        // Callbacks are disabled when transferring ownership.
+        loans[loanID_].callback = false;
```

Comment:
Is there no way to turn .callback back on after a transfer?

If failures in the callback did not cause overall reversion, then the only concern would be gas.

Reply:
since some auditoors pointed out that lenders can fake the caller callback support to force borrowers to default, we decided to only allow callbacks from trusted lenders (if they are the ones doing the loan request on behalf of the owner).

To prevent someone from looking legit, but then moving ownership to a malicious contract, we thought this was best.

Tbh, with the usage of `recipient` instead of `directRepay` I think that ownership changes doesn't matter that much anymore, so we are good with this tradeoff.

Reply:
You can just use .call() or .send(), and then this is a non-issue.